recipes-devtools/rust/rust-cross-canadian.inc

```diff
@@ -0,0 +1,224 @@
+
```

Comment:
I'm hitting some QA warnings during packaging:

```
WARNING: QA Issue: rust-cross-canadian-aarch64: found library in wrong location: /opt/sdk/sysroots/x86_64-pokysdk-linux/usr/lib/librustc_macros-d9145fa255675431.so
rust-cross-canadian-aarch64: found library in wrong location: /opt/sdk/sysroots/x86_64-pokysdk-linux/usr/lib/libtracing_attributes-301db28eaa169a2a.so
rust-cross-canadian-aarch64: found library in wrong location: /opt/sdk/sysroots/x86_64-pokysdk-linux/usr/lib/libstd-86e31077140c3cde.so
rust-cross-canadian-aarch64: found library in wrong location: /opt/sdk/sysroots/x86_64-pokysdk-linux/usr/lib/libchalk_derive-f58cadaeafa0e1a3.so
rust-cross-canadian-aarch64: found library in wrong location: /opt/sdk/sysroots/x86_64-pokysdk-linux/usr/lib/libtest-cd5cb2eef27bf3af.so
rust-cross-canadian-aarch64: found library in wrong location: /opt/sdk/sysroots/x86_64-pokysdk-linux/usr/lib/librustc_driver-7571c27e0d047d0f.so [libdir]
```

Reply:
Lets see if i can repro it, would need more data from your setup as per the request your first comment.

Thanks.

Reply:
Though please note - if your checks was on something lower than dunfell, please used dunfell first and lets move from there.

Thanks.